### ISSUE TITLE
Page in multiple languages

### ISSUE THREAD

**Issue by apohllo** [FlyoverWorks/storytime#146]
Do you plan to provide a feature for multilingual pages?

**Comment by robertwbradford** [comment]
+1

**Comment by bcroesch** [comment]
This is something we'd like to do at some point, but I can't honestly say that it's likely someone on our team will have time in the near future to work on it. If anyone is interested in taking it on and opening a PR, we'd certainly be open to it. Otherwise, we'll have to revisit when our team has a little more capacity.

**Comment by apohllo** [comment]
Ok, I will see what I can do. Maybe you can share some ideas how this should be implemented? That would help a lot.

**Comment by bcroesch** [comment]
Yep - let me think about potential implementation details for a day or two and will post them here.

**Comment by apohllo** [comment]
Sure. I'll be looking forward.

**Comment by apohllo** [comment]
Are there any chances to move this feature forward?

**Comment by mekhovov** [comment]
any updates?

**Comment by mekhovov** [comment]
by the way, in that fork https://github.com/Oliviergg/storytime @Oliviergg added multi-language support, can we use this approach?
